docs/src/index.md

```diff
@@ -25,10 +25,13 @@ A package for building documentation from docstrings and markdown files.
 
 The [Package Guide](@ref) provides a tutorial explaining how to get started using Documenter.
 
+A quickstart version of the guide can be found at [Quickstart]("https://github.com/JuliaDocs/Documenter.jl/blob/master/docs/src/quickstart.md")
```

Comment:
Hello @MichaelHatherly . Thank you for the review. :)
I felt that as documentation for someone who knows their way around these things it was perfect. But for someone just starting out with Julia/actually developing, it would be a bit hard to follow as many steps that were not related to Documenter.jl were involved. 
I had initially written a post on my blog but I was [told](https://discourse.julialang.org/t/step-by-step-documentation-tutorial/46093) that it might be a good addition to the documentation here so the PR.

I will fix whatever you mentioned by tomorrow. 
Cheers